### PR TITLE
update documentation to follow technical style guide

### PR DESCRIPTION
**Pull request by jankishah-gds**
## Tickets solved

What tickets will be solved (marked as done) once this PR is merged?

- [FPAD-XXX](https://govukverify.atlassian.net/browse/FPAD-XXX)

## Tickets created

What tickets have been created during this PR? Any bugs found? Out-of-scope features?

- [FPAD-XXX](https://govukverify.atlassian.net/browse/FPAD-XXX)

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging your code._

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] I have tested my changes on AWS

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, links to any design documents, etc...

### CODE REVIEW

File: receiver_guidance.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        "iss": "https://idp.example.com/",
+        "sub": "7375626A656374"
+      },
+      "reason": "hijacking"
+    }
+  }
+}
+```
+
+The claims are signed to ensure integrity and authenticity. This format is known as JSON Web Signature (JWS) and is the most common representation of the JWT. JWS has the format: `{HEADER}.{PAYLOAD}.{SIGNATURE}`
```
Comment (Cavalier3142) [inline receiver_guidance.md:50]: A JWT actually uses JWS for its signature. From the JWT spec:

JSON Web Token (JWT) is a compact, URL-safe means of representing claims to be transferred between two parties. The claims in a JWT are encoded as a JavaScript Object Notation (JSON) object that is used as the payload of a JSON Web Signature (JWS) structure or as the plaintext of a JSON Web Encryption (JWE) structure, enabling the claims to be digitally signed or MACed and/or encrypted.

So a JWT is actually a JWS structure with a JSON object as the payload.

File: receiver_guidance.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- the full URL of the /authorize endpoint, to request an authentication token
+
+You must send the SSF team:
+
+- the public key that corresponds with the private key used to sign the SET, so the SSF team can read the SET
+- the IP address that sends the request, so it can be added to their allowlist
+
+Send the SET to the receiver as a `POST` request to the SSF receiver endpoint. To assemble the request:
+
+1. Sign the SET using your private key to generate the signature. The signed SET and signature are used to create a JWS object. This will be the request payload.
```
[the remark this answers is not in the document]
Comment (jankishah-gds) [reply receiver_guidance.md:68]: The standard in the tech writing team is to number them all as 1. Then they will automatically have correct numbering even when things get moved around. Can write out as 1/2/3/4?

File: README.md [hunk 3]
```diff
@@ -1,163 +1,155 @@
 # The Relying Party Transmitter
```
[the remark this answers is not in the document]
Comment (jankishah-gds) [reply README.md:1]: Will check with Dan

File: receiver_guidance.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- [how to send a SET](#sending-a-set)
+
+## Understanding SSF concepts
+
+This application uses the following Shared Signals Framework (SSF) concepts:
+
+- transmitter - the entity responsible for broadcasting events
+- receiver - the entity responsible for receiving events
+
+A Relying Party (RP) is the transmitter of a Security Event Token (SET) and the SSF receiver endpoint is the receiver.
```
[the remark this answers is not in the document]
Comment (jankishah-gds) [reply receiver_guidance.md:16]: Is this accurate?: When a Relying Party (RP) sends a SET to the SSF, this is known as the inbound flow. In this case, the RP is the transmitter of a SET and the SSF receiver endpoint is the receiver. When the SSF sends a SET to the RP, this is known as the outbound flow. For the outbound flow, the SSF is the transmitter and the RP is the receiver.

File: receiver_guidance.md (hunk 1 above)
Comment (jankishah-gds) [reply receiver_guidance.md:50]: I was using this from the JWT RFC: "JWTs are always represented using the JWS Compact Serialization or the JWE Compact Serialization". I think I've now noticed the difference between JWS as the signature and JWS Compact Serialization as the format, which wasn't clear. 

I'll update this to: This format is known as JSON Web Signature (JWS) Compact Serialization format and is the most common representation of the JWT.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jankishah-gds) [reply README.md:1]: Dan had said that he's not aware of anything called EventBridge, so I will assume that it has been removed

[the remark this answers is not in the document]
Comment (Cavalier3142) [reply README.md:1]: Mmm I'm sure it is mock-RP. Used for scheduling. From the mock-RP readme:

_## 2.2. Eventbridge Scheduler

EventBridge is set to be used to activate the API endpoint on a configured schedule.

EventBridge will only operate on the main branch of each environment (such as Build, Integration, and Staging), each environment will have one corresponding EventRule. The EventBridge rule is defined within the `template-mock-rp.yaml` file amd will be responsible for managing the schedule and configuration of the `Generator Lambda`.

If the lambda is changed to handle additional properties they would need to be included in the event pattern to enable further configuration, they should be directly added to the `EventRule.Properties.EventPattern.Detail` object. The timing for publishing this event to the `Generator Lambda` target is determined and can be configured within the `EventRule.Properties.SchedulingExpression` object.

Any adhoc modifications to the EventBridge rule's event values are expected to be made through the AWS Dashboard. Keep in mind that permanent changes will only take effect when the branch is merged into the main branch. However, any adhoc changes will override these values._

[the remark this answers is not in the document]
Comment (jankishah-gds) [reply README.md:1]: Dan has said that the scheduler is disabled in the mock, but hasn't been included in the RPT

**Pull request merged**